design/features/editing-drafts/version-1.md

```diff
@@ -34,39 +34,39 @@ ___
 
 ### As-is
 
-- currently we take users to a new tab to preview their form as a form filler would see it  
-- there is a “DRAFT” watermark on the background of these screens, which we hope will help form creators keep the “creation” mindset (hat)  
-- there is no easy way for users to get back to the edit pages  
-- in rounds of research the new tab caused issues and users tried to click the back button to edit pages, with some being lost and needing prompting from the researcher  
-- some users suggested being able to quickly edit pages they were on when they saw an issue or error, like a typo or incorrect input type 
+- Currently, we take users to a new tab to preview their form as a form filler would see it  
+- There's a “DRAFT” watermark on the background of these screens, which we hope will help form creators to retain the 'creation' mindset (or 'hat')  
+- There's no easy way for users to get back to the edit screen of whichever question they're previewing  
+- During research the new tab caused issues as users tried to click the back button to edit the page they were viewing: some got lost and needed a prompt from the researcher  
+- Some users said they'd like to be able to quickly edit the pages they were on when they spotted an issue or error, like a typo or incorrect input type 
 
-#### Current preview screens design
+#### Current design for preview screens 
 
-The following designs were developed alongside the work to help users identify easily what version of the form they were looking at while the Forms team developed the new feature that allows a new draft of a form to be created from an existing live form without making changes instantly live.
+The following designs were developed alongside the work to help users easily identify which version of the form they were looking at. At the same time, the Forms team were developing the new feature that allows a new draft form to be created from an existing live form without making the changes instantly live.
```

Comment:
Hope this makes sense? (I wanted to break it up into two sentences for readability but wasn't 100% sure that I'd grasped your point correctly.)